README.md

```diff
@@ -1,13 +1,12 @@
-# Origami asset loader [![NPM version](https://img.shields.io/npm/v/@financial-times/health-check.svg)](https://www.npmjs.com/package/@financial-times/health-check) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
-
+# o-assets [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
 
 This component provides Sass and JavaScript utilities for reliably building paths to a component's static assets, such as CSS background images, fonts and JSON data files. This is needed because the URL path to load these assets will vary depending on how a product developer chooses to integrate the Origami component into their website.
 
 - [Overview](#overview)
-- [JavaScript](#javascript)
 - [Sass](#sass)
+- [JavaScript](#javascript)
 - [URL routing](#url-routing)
-- [Using with the build service](#using-with-the-build-service)
+	- [Using with the build service](#using-with-the-build-service)
```

Comment:
lovely

Reply:
yes! it complains:

![tooltip over this line of code showing: origami-component/readme-has-toc The table of contents items should be in the same order as the headings in the readme. Item "Using with the build service" does not match heading "Contact"](https://user-images.githubusercontent.com/178266/83515726-60eed980-a4c5-11ea-8cd1-eb66ca9adb2c.png)